roads.mss

```diff
@@ -1710,9 +1754,13 @@ come in as well.
         [type='bridleway'][can_bicycle='yes'] {
           line-width: @rdz17_path;
         }
-        [type='cycleway'] {
+        [type='cycleway'],
+        [type='path'][designated='yes'],
+        [type='footway'][designated='yes'],
+        [type='bridleway'][designated='yes'] {
           line-width: @rdz17_cycle;
-          [oneway='no'] { line-width: @rdz17_cycle*1.5; }
+          [oneway='no'][oneway_bicycle='no']
```

Comment:
We should probably try to avoid this double check here. I should review the exact tags we need, but probably we could fill in the `oneway_bicycle` value with the `oneway` value by default, therefore having:
* `oneway` which would be the general traffic oneway
* `oneway_bicycle` which would be the real oneway status for bikes